src/main/java/org/jenkinsci/plugins/github_branch_source/ApiRateLimitChecker.java

```diff
@@ -42,46 +42,48 @@ long checkRateLimitImpl(@NonNull GHRateLimit.Record rateLimit, long count, long
                             // work out how long until remaining == ideal + 0.1 * buffer (to give some spend)
-                            double targetFraction = (rateLimit.getRemaining() - buffer * 1.1) / (rateLimit
-                                .getLimit() - buffer - burst);
+                            double targetFraction = (rateLimit.getRemaining() - buffer * 1.1)
+                                    / (rateLimit.getLimit() - buffer - burst);
                             expiration = rateLimit.getResetDate().getTime()
-                                - Math.max(0, (long) (targetFraction * MILLIS_PER_HOUR))
-                                + ENTROPY.nextInt(1000);
+                                    - Math.max(0, (long) (targetFraction * MILLIS_PER_HOUR)) + ENTROPY.nextInt(1000);
                             writeLog(String.format(
-                                        "Jenkins-Imposed API Limiter: Current quota for Github API usage has %d remaining (%d over budget). Next quota of %d in %s. Sleeping for %s.",
-                                        rateLimit.getRemaining(),
-                                        ideal - rateLimit.getRemaining(),
-                                        rateLimit.getLimit(),
-                                        Util.getTimeSpanString(rateLimitResetMillis),
-                                        // The GitHubRateLimitChecker adds a one second sleep to each notification loop
-                                        Util.getTimeSpanString(1000 + expiration - now)));
+                                    "Jenkins-Imposed API Limiter: Current quota for Github API usage has %d remaining (%d over budget). Next quota of %d in %s. Sleeping for %s.",
+                                    rateLimit.getRemaining(),
+                                    ideal - rateLimit.getRemaining(),
+                                    rateLimit.getLimit(),
+                                    Util.getTimeSpanString(rateLimitResetMillis),
+                                    // The GitHubRateLimitChecker adds a one second sleep to each notification loop
+                                    Util.getTimeSpanString(1000 + expiration - now)));
                         }
                     }
                     if (expiration != now) {
-                        writeLog("Jenkins is attempting to evenly distribute GitHub API requests. To configure a different rate limiting strategy, such as having Jenkins restrict GitHub API requests only when near or above the GitHub rate limit, go to \"GitHub API usage\" under \"Configure System\" in the Jenkins settings.");
+                        writeLog(
+                                "Jenkins is attempting to evenly distribute GitHub API requests. To configure a different rate limiting strategy, such as having Jenkins restrict GitHub API requests only when near or above the GitHub rate limit, go to \"GitHub API usage\" under \"Configure System\" in the Jenkins settings.");
```

Comment:
Oof, this looks pretty bad. Maybe rules pertaining to line length (which I guess is responsible for this) can be suppressed?

Reply:
@jglick @jtnord 
I tried changing the method name to `log` and got:
```
                    if (expiration != now) {
                        log("Jenkins is attempting to evenly distribute GitHub API requests. To configure a different rate limiting strategy, such as having Jenkins restrict GitHub API requests only when near or above the GitHub rate limit, go to \"GitHub API usage\" under \"Configure System\" in the Jenkins settings.");
                    }
```

:shrug:  
Apparently, even one character improvement causes wrapping.  I take this as another case where the formatter is highlighting not great code by making it uglier.

If there were an option to `never wrap first argument`, I'd use it.  But there isn't so, I'll take this.

Reply:
> If there were an option to never wrap first argument,

comes back to auto formatters are not as smart as a good person.

Reply:
> > If there were an option to never wrap first argument,

> comes back to auto formatters are not as smart as a good person.

And [John Henry beat the steam engine](https://en.wikipedia.org/wiki/John_Henry_(folklore)), but I don't want to be John Henry.  😄

Reply:
This can be addressed by making the method name shorter or by manually splitting the string onto multiple lines.  I've opted for the latter.